### DISCUSSION TITLE
Prime Tower First Layer Speed and Spiral Z-Hop?

### DISCUSSION THREAD

**Discussion by Durahl** [OrcaSlicer/OrcaSlicer#2141] [Q&A]
So, I've downloaded the 1.7.0 version since it now seems to support Single Extruder Multi Material Printers like my Voron V2.4 equipped with an ERCF - ( without adding janky G-Code to the Tool Change Macro ).

**First Layer Speed:**
One thing I've come to notice is the slow as molasses speed of the first few Purge moves of the first Filament on the first Layer on Prime Tower ( blue ones in the sub 10mm/s range ):
![IMG](https://i.imgur.com/HzTMcu6.png)
What Setting is responsible for _that_ speed in particular **and** what setting is responsible for the remaining speed on the first Layer ( green one in the 100mm/s range )?

**Spiral Z-Hop:**
Even with Z-Hop Type set to `Normal` I see it doing `Spiral` ones ( containing G3 Commands producing errors ).

I'll have to see if enabling `[Gcode_Arcs]` to Klipper to deal with that will fix another issue I have with blobs forming at the beginning/end of every new Color printed after Priming that has propped up with this Version ( getting late to test now ) but in any case... Why is it doing Spiral Z-Hop when Normal is configured?

**Comment by TechSavage1989** [comment]
I'm having the same issue and it's breaking my post-processing scripts.
